### ISSUE TITLE
Mod "enabled" documentation vs. behaviour

### ISSUE THREAD

**Issue by JHCD** [chaorace/cqui#327]
> NOTE: Until this mod is in a less experimental state, the mod will not remember its "enabled" status after restarting the game. This means that every time you plan to start a new game using CQUI, you'll need to explicitly enable it in the "Additional Content" menu first. Once you've created the game using the mod, it will always load when you start the save, regardless of its current status in the "Additional Content Menu"

Is this still up-to-date?
It's always _enabled_ here.

**Comment by chaorace** [comment]
No, it's no longer an up to date description. I plan to just completely rewrite the readme for the Beta release

**Comment by JHCD** [comment]
We need a documentation of all used key bindings and mouse-click implementations.
Has anybody an overview about the possible features?

See #536 and #529 for example...
